btb/tuning/gcp.py

```diff
@@ -125,7 +127,12 @@ def jitter(x, range):
             return
 
         # -- Non-parametric model of 'y', estimated with kernel density
-        kernel_pdf = st.gaussian_kde(y)
+        try:
+            kernel_pdf = st.gaussian_kde(y)
+        except LinAlgError as e:
```

Comment:
maybe log this? I worry that LinAlgErrors could result from other conditions besides singular matrices and these bugs would be totally lost. Log on `debug` with manually printed stacktrace or log using `exception` level with automatically printed stacktrace, depending on your judgment?